Server/controllers/user.controller.js

```diff
@@ -1,17 +1,66 @@
-"use strict";
+const datalayerFactory = require('../datalayer/factory.datalayer');
+const userDataLayer = require('../datalayer/user.datalayer');
 /**
  * Class representing users controller
  *
  * @class usersController
  */
 class usersController {
 
-  static createUser (req, res) {}
+  static async createUser (req, res) {
+    let reqObject = req.body;
+        try {
+            await userDataLayer.createUser(reqObject);
+        } catch(err) {
+            res.status(400).send(err.message);
+            return;
+        }
+        res.status(200);
+        res.send("User created successfully");
+  }
 
-  static getUsersByUserId (req, res) {}
 
-  static updateUserById (req, res) {}
+  static async getUsersByUserId (req, res) {
+    let userid = req.params._id;
```

Comment:
What is URL for this route for my reference?

Reply:
URL for this route is : api/v1/users/:id

Reply:
change it to `api/v1/users/:userid` and update controller code

Reply:
@ashmeetkaur-compro please update this using and update controller code

Reply:
Replaced _id with userid